moduleroot/Gemfile.erb

```diff
@@ -1,22 +1,38 @@
 <%
 def gem_length(gem)
-  if gem['from_env']
-    version_len = " *location_for(ENV['#{gem['from_env']}'])".length + ((" || '#{gem['version']}'".length if gem['version']) || 0)
+  if gem['version']
```

Comment:
There is some code duplication here with the function defined on line 21. I wonder if we could reduce this and share  some commonalities?

Reply:
Not without redoing a good bit of the template, the logic on how it works is a bit backward.

Reply:
Fair enough. Something that we can look at in the future then.